airflow/include/gcp/service_account.json

```diff
@@ -0,0 +1,13 @@
+{
+  "type": "service_account",
+  "project_id": "my-new-gcp-project-406521",
+  "private_key_id": "c5925f033b0d0d8e59dfd63f8f6ed46614363946",
+  "private_key": "-----BEGIN PRIVATE KEY-----\nMIIEvAIBADANBgkqhkiG9w0BAQEFAASCBKYwggSiAgEAAoIBAQDTfCJ6fR2mILJq\n0W8Az9MiBcpFiBwgW9ncolNT4aO/KT7bnFOFKDO7ywVBdK2hPnD4cKQqfKLnRNg6\nQN+XzSTtMQw1oIQCj5OmRkWygl2j1qpQ2iWo46jaC0aABiwP2Ab7C652Or5Hqw9y\nTnFNV03J3iidwgNiTr17i4r2mQ/BT2dlUFeI0XuEdif0biS/utAghvQt6BRyPaOZ\n4kd384V9PqgifFFWba+7vXxXSh3D4r10eSQLi3RS6PgLttLVlW64kMUoSG4d6fQC\n9pDwibL1w4MEOt4H0KV7BiqYAuPxoQGxluwWHPjASXWD+26MnnsLc/hchy//m5by\nXsPPk6VNAgMBAAECggEASHYU/jEWUpCLcUwXI4kRx2uCdbaAJJo0GIPJ10jNqTcB\noY97knlG2IZkY5wNzdyuKoG1zhTsYB9XkwYdVk7k4xxWGZ3I/XOiXGthgBto6DUq\nIA/OrYJrnWIVotH6kOcs5SGIglv4TGgRH9I+wggy0AL9magbLrjZ9ZXHZFznaA0k\nMKjB30hjnlB32xsDF5V5skxWpO1zj8J0SkdA9JbEGkKV+ZRukQkOzhsOf/ZbxIY+\n8q7TDYxCFweOL/hLxMkavEdlV1ctC0LKxS+ta8AM+oNOydO0X3hZ+aA28GnVWVGE\nb9e4K7YiSGaQMEI+SzqXXa7PBSY+GqmU4iskDjYoBQKBgQD05xsNo2CkLNN+FRu4\n6oAsNzvSd6QJ+3HrbFVaphTk+M6yGCCKFLuAukj92Ml+tmalvEdlj2xsDx/Mwhb5\nWAIAxTVi5ewirqF+YOoBN6ZLAJgyewBkKhXGrMW5jSvdqGIfhcSr8GstH9XzDICe\nm6mTf+KtoZXkf2TThmW1YDsGHwKBgQDdEWD7I/4nCScf/Lm+DvpZZlK/eFwLf8SS\nGmFCGRHGBmJ7oWxF9/7doGF1q32gQzFVa5XDreSBfsmw6GUPTOKuLVa8mbSzp2Ax\n9Zw2k4MHV78qTvFrOc3eueNcXa3cGiyWnY07N5/IVlllRZjnS68L3UMZRmcg4AoU\nCcGxT52vEwJ/T2aeXs2Zh6EA8N/nHQzqgWtm+dXQWWwsgyechMdT/uisGxG3ew09\nvX6oK6Udrvd0UEe63jKh9Gby6HAFAtKF07oclPP2m0gDE/ZX6A0i3S5UKgm3FsKO\nTNu3P3GFx1fQeGfmW9iW5wprU2G6UbImSKwt0h2Re4PSLMJmN2f7uwKBgQCcO2f6\nlNlM5R8NcCIku5kaDUJ1pq4GQmm9h38so2L4PBLlpJNMb5NjNbEq4+OIm3QGFl5l\nSEOxcNVBfTvOsFuTFN+zsj0Oqj9Fj3qy51U2n0CuudPMH7HqshyIbnwsjSLt3rKt\nIwZ9IulStPj2nXo8PEbrWeIQTcVRnAOTN90MmQKBgQCtmmfexQwdKFwj/OEziKbd\n/meY9ZImpbFcayaW6lNn1BOGcSH0dUnVwUMdpcKUiahUIR88f8xE9y1fjCNCGyGD\nzv5Zn2nouyZucvTEVLNnPDQ9qRFHcMc4tGxzYeq74GfPmrl+vyY3yBQqyMrGGqHd\nSE2kHv1/bHXyhebW5hUL5g==\n-----END PRIVATE KEY-----\n",
```

Comment:
please do not commit any private key or service account key, add this path to .gitignore, check how to remove it from git history

Reply:
delete this key (since it is a public repository) and use a new one but don't commit it in repo

Reply:
I just pressed delete but deactivate it asap, otherwise somebody could use the key and make your billing nightmare